Snowflake.Data.Tests/UnitTests/Logger/EasyLoggerManagerTest.cs

```diff
@@ -22,9 +22,7 @@ public class EasyLoggerManagerTest
         private const string ErrorMessage = "Easy logging Error message";
         private const string FatalMessage = "Easy logging Fatal message";
         private static readonly string s_logsDirectory = Path.GetTempPath();
-
-        [ThreadStatic]
-        private static string t_directoryLogPath;
+        private string _directoryLogPath;
```

Comment:
what was wrong with the previous form of thread static? It didn't work?

Reply:
There is no need to use static field so I removed it. Its because of: https://github.com/snowflakedb/snowflake-connector-net/actions/runs/6585609942/job/17892373465#step:8:4582

Reply:
This means that this test might be flaky. How your change is going to improve the situation? I think that the cause of the error is that we try to remove working directory after the test while logger has not flushed its logs to the file yet - or something like that. I don't see how your change will improve the situation.

Reply:
_The process cannot access the file 'C:\Users\runneradmin\AppData\Local\Temp\easy_logging_logs_dkdsyynl.bgd\dotnet\snowflake_dotnet_k0201fjm.log' because it is being used by another process._
The problem might be connected with static field. Since there is no need to use static I changed it to private.